bundles/ch.elexis.core.ui/src/ch/elexis/core/ui/dialogs/KontaktExtDialog.java

```diff
@@ -43,163 +40,141 @@
  * 
  */
 public class KontaktExtDialog extends TitleAreaDialog {
-	private Kontakt k;
-	private String[] f;
-	
-	public KontaktExtDialog(Shell shell, Kontakt k, String[] defvalues){
+	private String[] fieldDefinitions;
+	private Kontakt actKontact;
+	private ExtInfoTable infoTable;
+	/*
+	 * private HashMap<String, String> xids; private String[] fields; private Text[]
+	 * values;
+	 */
+	private static final org.slf4j.Logger logger = LoggerFactory.getLogger(KontaktExtDialog.class);
+
+	public KontaktExtDialog(Shell shell, Kontakt k, String[] defvalues) {
 		super(shell);
-		this.k = k;
-		f = defvalues;
-		Arrays.sort(f, new Comparator<String>() {
-			
-			public int compare(String o1, String o2){
+		this.actKontact = k;
+		logger.info("{} fields are: {}", defvalues.length, defvalues);
```

Comment:
Don't use logging this way - it might cause sensitive information to get into the logs! Also I don't see any reason to log this with level info